src/main/java/eu/bbmri_eric/negotiator/service/DiscoveryServiceClient.java

```diff
@@ -0,0 +1,10 @@
+package eu.bbmri_eric.negotiator.service;
+
+public interface DiscoveryServiceClient {
+
+  public void syncAllOrganizations();
+
+  public void syncAllResources();
+
+  public void syncAllNetworks();
```

Comment:
please add javadoc

Reply:
fixed in fb760484e9b2dbec0c0c5b584decfbd60305ad4d